fstapi/src/reader.rs

```diff
@@ -239,6 +239,41 @@ impl Reader {
       _ => Err(Error::InvalidOperation),
     }
   }
+  /// Gets the value of the variable with the given handle at the specified time.
+  ///
+  /// If the variable is not found, returns `None`.
+  /// If the value is not found, returns `None`.
+  pub fn get_value_from_handle_at_time(&mut self, time: u64, handle: Handle) -> Option<String> {
+    let mut buf = vec![0; 1024]; // Allocate a buffer for the value
+    let result = unsafe {
+      capi::fstReaderGetValueFromHandleAtTime(self.ctx, time, handle.into(), buf.as_mut_ptr())
+    };
+
+    let value_str = unsafe { std::ffi::CStr::from_ptr(result as *const raw::c_char).to_str().ok()? };
```

Comment:
* The `result` should be checked, since `fstReaderGetValueFromHandleAtTime` may return a null pointer.
* The return type of this method should be `Box<[u8]>` instead of `String`, to be consistent with the writer API of this Rust crate.